src/lib/snarkette/fields.ml

```diff
@@ -135,7 +141,24 @@ module Make_fp
 
   let ( * ) x y = x * y % Info.order
 
-  let square x = x * x
+  let square x = x * x % Info.order
```

Comment:
Isn't this `%` redundant? The definition of `*` right above includes it.

Reply:
Yes thank you!